### DISCUSSION TITLE
Keyball44について、1_左手の特定キーが反応しない、2_スペーサーを挟まず、直接キースイッチをハメると認識する件

### DISCUSSION THREAD

**Discussion by sbsyou** [Yowkees/keyball#493] [Q&A] [answered]
Keyball44について、2件質問がございます。

1_左手の特定キーが反応しない
赤枠で囲ったキーが反応しません。対応するダイオードなどございましたら、
ご共有いただくこと可能でしょうか。

2_スペーサーを挟まず、直接キースイッチをハメると認識する
緑枠で囲まれたロープロファイル3件につきまして、
説明書通り、スペーサーを挟むと反応しないのですが、
スペーサーを挟まず直接、キースイッチをはめると反応します。
このままでも使えそうですが、考えられる要因、対策などございますでしょうか。
![IMG_6553](https://github.com/Yowkees/keyball/assets/158572235/ea235286-dc56-4a6a-a8db-a251b74248ab)

参考まで、pro microアップ写真も添付します。
![IMG_6554](https://github.com/Yowkees/keyball/assets/158572235/af42c586-6506-4fe4-8328-ddb7e13a7779)

よろしくお願いします。

**Comment by sbsyou** [comment, marked as answer]
自己解決したので、追記します。
キーなどが反応しない件、まわりのダイオード、ソケットの半田付けができていないと判断し、フラックスなどつけて再度ハンダして対応。それでも反応しないところは、キースイッチの足が曲がっていて通電できていない箇所があったので、修正。

ロープロファイルの件はそもそもその設定だと認識し解決しました。

これらの対応で無事改善できました。全てのキーボードが反応し、keyball44ライフをこれから楽しみます！
ありがとうございました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

ご自身で解決いただけたようで良かったです。
ぜひ末永くKeyball44をご愛用いただければ幸いです。今後ともどうぞよろしくお願いいたします。
